wp/lib/bap_wp/tests/unit/test_compare.ml

```diff
@@ -425,6 +425,43 @@ let test_sub_pair_fun_4 (test_ctx : test_ctxt) : unit =
     compare_prop Z3.Solver.SATISFIABLE
 
 
```

Comment:
Why can't we use BIL to BIR here?

Reply:
Because I didn't think. Using Bil to Bir will definitely be cleaner here.